aiohttp/multipart.py

```diff
@@ -980,14 +980,14 @@ def size(self) -> Optional[int]:
         """Size of the payload."""
         total = 0
         for part, encoding, te_encoding in self._parts:
-            if encoding or te_encoding or part.size is None:
+            if encoding or te_encoding or (part_size := part.size) is None:
```

Comment:
I think this probably makes sense on a separate line.
```suggestion
            part_size = part.size
            if encoding or te_encoding or part_size is None:
```